CMakeLists.txt

```diff
@@ -55,5 +55,11 @@ include(CppCheck)
 include(ClangTidy)
 include(ClangFormat)
 include(Doxygen)
-  
+
+add_subdirectory(TASTE-Linux-Runtime/src)
+add_subdirectory(TASTE-Runtime-Common/src/RuntimeCommon)
```

Comment:
you could just `add_subdirectory(TASTE-Runtime-Common/src)`

Reply:
Yes, that is true. At first, I wanted only to include those parts of Runtime-Common that are needed, but I ended up adding all of src. 
Now I've realized that I don't have to include RuntimeCommon. Is it worth it to delete it and keep all of the other paths, or should I include all src as the compilation time is short anyway, so it won't hurt to compile not-needed static libraries?

Reply:
If there is something not needed in the `src` directory then you can leave this as is to just build what is actually used